src/tools/devtools/release-notes/release-notes-2.22.2-src.md

```diff
@@ -41,6 +41,6 @@ in deeply nested trees - [#5181](https://github.com/flutter/devtools/pull/5181)
 ## Network profiler updates
 - Improve reliability and performance of the Network tab - [#5056](https://github.com/flutter/devtools/pull/5056)
 
-## Changelog
-More details about changes and fixes are available in the DevTools
-[changelog](https://github.com/flutter/devtools/blob/master/CHANGELOG.md).
+## Full commit history
+More details about changes and fixes are available from the
+[DevTools git log.](https://github.com/flutter/devtools/commits/master).
```

Comment:
**issue:** Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure). Write in active voice per [GSG](https://developers.google.com/style/voice). Use site variable.
```suggestion
To learn more about DevTools changes and fixes, check out the
[commit history]({{site.github}}/flutter/devtools/commits/master).
```